web/src/message_edit.js

```diff
@@ -348,6 +348,7 @@ function handle_message_row_edit_keydown(e) {
                     return;
                 }
                 save_message_row_edit($row);
+                $(document).trigger("message_editing_done.zulip");
```

Comment:
I think it's likely better for this sort of trigger to be inside `save_message_row_edit` -- it's a bad pattern to split logic between the click handler and the function that actually does stuff -- and in particular will create bugs where this event is sent incorrectly if `save_message_row_edit` fails without throwing an exception.